### PR TITLE
Patch PCP mobile

### PR DESCRIPTION
**Pull request by AaDalal**
This PR accomplishes the following:
- Restyles tabs
- Rescales UI to fit mobile
  - Increase search bar size
  - increase font size 
- Makes other small visual changes

This PR is a stopgap to help make PCP mobile more usable.

TODO:
- [x] add back PCP logo
- [x] ~~change location of filters~~
- [x] force switch to search tab when search bar is clicked
- [x] Test schedule tab (with courses that have times associated)
- [x] ~~Fix weird multiple font issue~~

### CODE REVIEW

File: frontend/plan/components/search/SearchBar.tsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import styled from "styled-components";
 // @ts-ignore
 import AccountIndicator from "pcx-shared-components/src/accounts/AccountIndicator";
 import { useRouter } from "next/router";
+import * as ReactDOM from "react-dom";
+import dynamic from "next/dynamic";
```
Comment (andyjiang3) [inline frontend/plan/components/search/SearchBar.tsx:9]: Are these unused imports? If so, can you remove them

File: frontend/plan/components/schedule/Schedule.tsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ const ScheduleBox = styled.div`
     color: #4a4a4a;
     display: block;
     padding: 1.25rem;
-    height: calc(100vh - 12em);
+    min-height: calc(100vh - 12em);
 `;
```
Comment (andyjiang3) [inline frontend/plan/components/schedule/Schedule.tsx:89]: Seems like adding a course to the schedule expands the schedule box height, which makes the height of the boxes unequal. Why do we need `min-height` here?

Screen Shot 2022-09-06 at 8 32 53 PM

**Review by andyjiang3 — COMMENTED** [review]
Great job resolving the mobile issues! Overall it looks good, just some comments! 

Also, the fonts and icons look kinda small on mobile view (might be because im looking at it on a laptop), maybe thats something we can revisit later cuz seems like we need to scale everything up a bit on mobile. 

Screen Shot 2022-09-06 at 8 37 19 PM

File: frontend/plan/components/search/SearchBar.tsx (hunk 1 above)
Comment (AaDalal) [reply frontend/plan/components/search/SearchBar.tsx:9]: Yup, good catch.

File: frontend/plan/components/schedule/Schedule.tsx (hunk 2 above)
Comment (AaDalal) [reply frontend/plan/components/schedule/Schedule.tsx:89]: min height allows for expansion on mobile, where the viewport height may not be sufficient to hold the stats at the bottom. I'll create a new styled component for mobile or add a media query.

**Review by andyjiang3 — APPROVED** [review]
LGTM!

**Pull request merged**